fizzbuzz.py

```diff
@@ -0,0 +1,20 @@
+#function fizzbuzz
+def fizzbuzz(a, b):
+    if isinstance((a and b), list):
+        combined_length = len(a) + len(b)
+        
+        if ((len(a) + len(b)) % 3 == 0) and ((len(a) + len(b)) % 5 == 0):
+            return "fizzbuzz"
+        elif (len(a) + len(b)) % 3 == 0:
+            return "fizz"
+        elif (len(a) + len(b)) % 5 == 0:
+            return "buzz"
+        else:
+            message = "The length of these lists isn't divisible by both 5 and 3, and it's:"
```

Comment:
You are creating a variable called `message` and you are not using it anywhere

Reply:
Oh thanks @dennisja , let me just remove it